### ISSUE TITLE
[Feature] Allow both webform and OIDC authentication

### ISSUE THREAD

**Issue by onnyyonn** [FreshRSS/FreshRSS#6498]
I am planning to set up a FreshRSS instance for a few users, some of whom would like to have OIDC authentication, and some would prefer webform. It seems, I can't have both at the same time. Some other applications (e.g., Nextcloud) allow having both. Will that be possible in FreshRSS?

**Comment by Alkarex** [comment]
No, but the easiest for now would be to spin two FreshRSS instances

**Comment by pippo73** [comment]
It's a pity you closed this request that is reasonable.
